src/java.base/share/classes/sun/util/calendar/CalendarSystem.java

```diff
@@ -120,7 +120,17 @@ private static void initNames() {
      * @return the <code>Gregorian</code> instance
      */
     public static Gregorian getGregorianCalendar() {
-        return GREGORIAN_INSTANCE;
+        var gCal = GREGORIAN_INSTANCE;
```

Comment:
Do we need the local variable `gCal`?

Reply:
This was there to avoid additional volatile reads in that method. A performance optimization. However, with the change Roger suggested, this is no longer relevant.